Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG node_version=16-alpine3.13
```

Comment:
I ARG here. This should be passed in with ``--build-arg NODE_VERSION=***``
Also capitalise for styling.